components/carousel/private/auto-play-button.jsx

```diff
@@ -0,0 +1,54 @@
+/* Copyright (c) 2015-present, salesforce.com, inc. All rights reserved */
+/* Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license */
+
+// ### React
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { CAROUSEL_AUTOPLAY_BUTTON } from '../../../utilities/constants';
+
+/**
+ *  AutoPlayButton is used to start/pause the autoplay iteration of the carousel
+ */
+const AutoPlayButton = (props) => {
+	const icon = props.isAutoPlayOn
+		? '/assets/icons/utility-sprite/svg/symbols.svg#pause'
+		: '/assets/icons/utility-sprite/svg/symbols.svg#play';
+
+	return (
+		<span className="slds-carousel__autoplay">
+			<button
```

Comment:
The `ButtonIcon` component should be used here, other context globals like Icon paths will be used and will be hard-coded. See `import ButtonIcon from '../icon/button-icon';` 
```
<ButtonIcon
    category="utility"
    name="down"
    size="x-small"
/>
```
if we didn't want a button appearance. A standard button would work if we want a gray background.

Reply:
done